src/components/FeatureViews/Scheduler/ScheduleForm.js

```diff
@@ -0,0 +1,130 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as calenderActions from '../../../actions/calenderActions';
```

Comment:
Redux does not seem to be used in the component, remove it (and associated `mapStateToProps`, `mapDispatchToProps`, and `connect` at the bottom).